backend/test/image/ImageRepositoryTest.ts

```diff
@@ -0,0 +1,31 @@
+import { should } from 'chai';
+import Image, { ImageDoc } from '@src/image/Image';
+import { common as commonTestData } from '@test/data/testData';
+import { ClientSession, Connection, CreateQuery } from 'mongoose';
+import ImageRepository from '@src/image/ImageRepository';
+import { getConnection, setConnection } from '@src/common/mongodb/DbConnectionUtil';
+import { abortTestTransaction, replaceUseTransactionForTest } from '@test/TestUtil';
+
+describe('ImageRepository test', () => {
+  let imageRepository: ImageRepository;
+  before(async () => {
+    should();
+    imageRepository = new ImageRepository();
+    setConnection();
+    await replaceUseTransactionForTest();
+  });
+
+  afterEach(() => {
+    const conn: Connection = getConnection();
+    conn.close();
+  });
+
+  it('createImages', async () => {
+    const imageCreateQueryList: CreateQuery<ImageDoc>[] = [commonTestData.gifImage, commonTestData.pngImage];
+    // @ts-ignore
+    const session: ClientSession = await imageRepository.createImages(imageCreateQueryList);
```

Comment:
If a repository function has a return value, this won't be a good way to get session value.
Find another way to get session from the fake function.